src/components/NewMessage/NewMessage.vue

```diff
@@ -506,6 +506,12 @@ export default {
 				}
 			}
 
+			// FIXME upstream: https://github.com/nextcloud-libraries/nextcloud-vue/issues/4492
+			const temp = document.createElement('textarea')
+			temp.innerHTML = this.text.replace(/&/gmi, '&amp;')
```

Comment:
Not sure if text couldn't be empty or undefined here

Reply:
it shouldn't be undefined in any case but we can add a condition to be not empty ( '' )